pub-kube-config-files/kafka-lagcheck.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: kafka-lagcheck
+  labels:
+    app: kafka-lagcheck
+    visualize: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kafka-lagcheck
+  template:
+    metadata:
+      labels:
+        app: kafka-lagcheck
+        visualize: "true"
+    spec:
+      containers:
+      - name: kafka-lagcheck
+        image:  coco/kafka-lagcheck:v1.0.15-k8s
+        imagePullPolicy: Always
+        resources:
+          limits:
+            memory: 256Mi
+        env:
+        - name: WHITELISTED_TOPICS
+          value: "Concept"
+        - name: BURROW_URL
+          value: "http://burrow:8080"
+        - name: LAG_TOLERENCE
```

Comment:
typo LAG_TOLERENCE -> LAG_TOLERANCE (verified in codebase and LAG_TOLERANCE is used)